application/src/main/java/uk/nhs/hee/tis/revalidation/messages/RabbitMessageListener.java

```diff
@@ -107,4 +108,20 @@ public void receiveUpdateMessageFromMasterDoctorView(
         recommendationViewMapper.mapMasterDoctorViewDtoToRecommendationView(
             masterDoctorViewDto));
   }
+
+  /**
+   * disconnect Doctors from a Designated Body (DB) when the GMC give us a DB without them.
```

Comment:
This method only handles the message, I think this description belongs with the service method 

Also might want to rename this method to something like `handleDoctorsForDbCollectedMessage`